app/views/news_tematica/news_tematicas/index.html.haml

```diff
@@ -2,11 +2,13 @@
 
 - unless @news_tematicas.blank?
   %table.ranking
-    = ths(%w(Título Temática fecha_envío enviada))
+    = ths(%w(Título Temática fecha_envío enviada -))
     - @news_tematicas.each do |news_tematica|
       %tr{ class: news_tematica.enviada ? 'par' : nil }
         %td= link_to news_tematica.titulo, edit_news_tematica_path(news_tematica)
         %td= news_tematica.nombre
         %td= l news_tematica.fecha_envio, format: :short
         %td.center= news_tematica.enviada ? 'Sí' : 'No'
+        %td= link_to('Borrar', news_tematica_path(news_tematica), method: :delete, data: { confirm: "¿Borrar la newsletter #{ news_tematica.titulo }?" })
 
+= will_paginate @news_tematicas
```

Comment:
You could add this pagination inside the `- unless @news_tematicas.blank?` block